packages/cactus-test-plugin-htlc-eth-besu-erc20/package.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "@hyperledger/cactus-test-plugin-htlc-eth-besu-erc20",
+  "version": "0.3.0",
+  "description": "Integration tests for the Besu ledger and the API server.",
```

Comment:
@jordigiam I'd rephrase the description a little bit to mention the ERC20 HTLC plugin

Reply:
Done!